ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class PostCommand extends IronfishCommand {
+  static description = `Post a raw transaction`
```

Comment:
Can you be specific on what will happen when post a raw transaction. Also include the instruction on offline signing here, like what bitcoin does for the examples in https://developer.bitcoin.org/reference/rpc/sendrawtransaction.html#examples

Reply:
I added docs. I think you asked me to add some docs here on how to get the raw transaction but this is opened before those commands exist, so that's not really possible. Let's add those docs in the PR for when we can get a raw tranaction.